tests/unit/test_response_utils.py

```diff
@@ -24,6 +27,40 @@ def test_when_processing_ok_response_return_response_object_without_error(self):
         self.assertEqual(output_response.status, HTTPStatus.OK)
         self.assertEqual(output_response.json, response_text)
 
+    def test_when_processing_plain_error_message_returns_parsed_message(self):
+        input_response = Mock(spec=Response)
+        input_response.status_code = HTTPStatus.REQUEST_TIMEOUT
+        error_message: str = '{"message":"Something went wrong."}'
+        input_response.json.return_value = json.loads(error_message)
+        input_response.text = error_message
+
+        assert_that(self.response_utils.handle_response)\
+            .raises(RestErrorException)\
+            .when_called_with(input_response)\
+            .starts_with('(\'Something')\
```

Comment:
I would prefer using outer double-quotes. The backslashes inside strings can be noisy.

Reply:
Thanks for spotting that. I am going to fix that and push, but won't ask another approval, just merge it.